src/ui/core/zcl_abapgit_html.clas.abap

```diff
@@ -189,7 +191,9 @@ CLASS zcl_abapgit_html IMPLEMENTATION.
         OR ls_study-tag_close = abap_true )
         AND cs_context-indent > 0.
       lv_spaces = ( cs_context-indent - 1 ) * c_indent_size.
-      cv_line  = gv_spaces(lv_spaces) && cv_line.
+      IF lv_spaces <= c_max_indent.
+        cv_line  = gv_spaces(lv_spaces) && cv_line.
+      ENDIF.
```

Comment:
```abap
ELSE.
  cv_line = gv_spaces && cv_line.
ENDIF.
```

Reply:
ahh yea, this would break more than fix, code updated